docker-compose/obkm/scripts/README.md

```diff
@@ -0,0 +1,78 @@
+# WSO2 Open Banking API Manager with Open Banking Key Manager
+
+
+## Prerequisites
+
+ * Install [Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git), [Docker](https://www.docker.com/get-docker) and [Docker Compose](https://docs.docker.com/compose/install/#install-compose)
+   in order to run the steps provided in following **Quick Start Guide**. <br><br>
+ * In order to use WSO2 Open Banking Docker images, you need an active WSO2 Open Banking subscription. If you do not possess an active WSO2
+   Open Banking subscription, you can contact us from [here](https://wso2.com/solutions/financial/open-banking/).<br><br>
+ * If you wish to run the Docker Compose setup using Docker images built locally, build Docker images using Docker resources available from [here](../../dockerfiles/) and remove the `docker.wso2.com/` prefix from the `image` name in the `docker-compose.yml`. <br><br>
+    
+## Quick Start Guide
+
+1. Clone WSO2 Open Banking Docker Git repository.
+
+    ```
+    git clone https://github.com/wso2/docker-open-banking.git
+    ```
+    > If you are to try out an already released zip of this repo, please ignore this 1st step. 
+
+2. Switch to the `docker-compose/obam-with-obkm` folder.
+
+    ```
+    cd docker-open-banking/docker-compose/obam-with-obkm
+    ```
+    > If you are to try out an already released zip of this repo, please ignore this 2nd step also. 
+     Instead, extract the zip file and directly browse to `docker-open-banking-<released-version-here>/docker-compose/obam-with-obkm` folder. 
+     
+    > If you want to try out an already released tag, after executing 2nd step, checkout the relevant tag, 
+     i.e. for example: git checkout tags/v1.4.0.5 and continue below steps.
+
+3. Execute the `deploy.sh` script to start the deployment.
+   ```
+   ./deploy.sh
```

Comment:
@Akila94 I suppose you are prompted to provide the WSO2 Subscription credentials when using this script.

It would be better if we can add a note instructing the user to provide the credentials of the WSO2 Subscription.

If this has not been added in other deployment patterns (i.e. in the automated `scripts`), please update them.

Reply:
It is added to the script files already. When executing the script, it prompts asking WSO2 username and password. The customer will eventually know when to enter the credentials because it prompts.

Reply:
@Akila94 yes, but still it would be better to add a note, mentioning this step is there.

Reply:
okay, will add.